conda-forge.yml

```diff
@@ -4,3 +4,6 @@ github:
 conda_build:
   error_overlinking: true
 conda_forge_output_validation: true
+bot:
+  automerge: true
```

Comment:
```suggestion
  automerge: true
  check_solvable: true
```

Reply:
As discussed in https://github.com/conda-forge/qiskit-terra-feedstock/pull/28#issuecomment-1699571043

Reply:
Ahh yes forgot that part. 😄